bodyfat.py

```
@@ -1,158 +1,19 @@
-    )
-    neck_cm: Optional[float] = Field(
-        None, gt=0, description="Neck circumference in cm, must be positive"
-    )
-    waist_cm: Optional[float] = Field(
-        None, gt=0, description="Waist circumference in cm, must be positive"
-    )
-    hip_cm: Optional[float] = Field(
-        None, gt=0, description="Hip circumference in cm, must be positive"
-    )
-    language: Optional[str] = "en"  # "en" | "ru"
+from core.bodyfat import bf_deurenberg, bf_us_navy, bf_ymca, estimate_all
```

Flake8 (7.3.0)

[error] 'core.bodyfat.estimate_all' imported but unused

(F401)